net-im/skype-dbus-mock/skype-dbus-mock-0_p20181011.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6,3_7} )
+
+inherit python-r1
+
+EGIT_COMMIT="3a9e2882ac5c0ad6be3c5cb5c7da008b4cfa51da"
+DESCRIPTION="Mocked systemd dbus interface for skype 8.30+"
+HOMEPAGE="https://github.com/maelnor/skype-dbus-mock"
+SRC_URI="https://github.com/maelnor/${PN}/archive/${EGIT_COMMIT}.tar.gz -> ${P}.tar.gz"
```

Comment:
1.) take a snapshot of the master branch, not a commit

Reply:
Are you suggesting that this would potentially pull a different git commit every time it's emerged (i.e. a 9999 ebuild)? So anything without a version would never be stabilized?